notebook2/environment.yml

```diff
@@ -9,3 +9,5 @@ dependencies:
   - Flask_Sockets
   - libsass
   - authlib
+  - PyJWT
```

Comment:
we really ought to be fixing major versions of our dependencies

Reply:
Done for all deps here: https://github.com/hail-is/hail/pull/5741